indico/modules/events/management/templates/action_menu.html

```diff
@@ -13,9 +13,14 @@
             <button class="i-button icon-lock-center js-ajax-dialog-lock" data-href="{{ url_for('event_management.lock', event) }}"
                     title="{% trans %}Lock event{% endtrans %}" {{ 'disabled' if event.isClosed() }}></button>
         {% endif %}
-        <button class="i-button icon-remove js-ajax-dialog-delete danger light" data-qtip-style="error"
+        <button class="i-button icon-remove js-ajax-dialog-delete danger light {{ 'disabled' if not can_manage }}"
                 data-href="{{ url_for('event_management.delete', event) }}"
-                title="{% trans %}Delete event{% endtrans %}"></button>
+                {% if can_manage %}
+                    title="{% trans %}Delete event{% endtrans %}"
+                    data-qtip-style="error"
+                {% else %}
+                    title="{% trans %}Only the event and category managers may delete an event.{% endtrans %}"
```

Comment:
"Only managers may delete this event"? Usually we only talk about "managers" without mentioning that category manages have full event management access. For locking we just mention it because they have more rights than an event manager